index.html

```diff
@@ -14,7 +14,7 @@
 
         <div class="page-heading">How Istanbul works</div>
         <p>
-          Istanbul instruments your ES5 and ES6 JavaScript code with line counters,
+          Istanbul instruments your ES5 and ES2016+ JavaScript code with line counters,
```

Comment:
I'm guessing you're joking, but I can't tell.... This text should be changed to `ES2015+`